font-codegen/Cargo.toml

```diff
@@ -13,16 +13,16 @@ path = "src/main.rs"
 
 [dependencies]
 font-types = { version = "0.0.5", path = "../font-types" }
-rustfmt-wrapper = "0.1"
+rustfmt-wrapper = "0.2"
 regex = "1.5"
-miette = { version =  "4.6", features = ["fancy"] }
+miette = { version =  "5.0", features = ["fancy"] }
 syn =  { version = "1.0", features = ["parsing",  "extra-traits", "full"] }
 proc-macro2 =  { version = "1.0", features = ["span-locations"]}
 quote = "1.0"
 toml = "0.5"
 serde = {version = "1.0", features = ["derive"] }
 xflags = "0.2.4"
 log = "0.4"
-env_logger = "0.9.0"
```

Comment:
Updating unrelated deps seems odd, maybe should have been two "just merge" grade PRs?

Reply:
yea, could've been. Basically i updated the miette dep (which handles error reporting) to see if this bug had been fixed, and then figured i'd do the others while I was in there.